docs/example.rst

```diff
@@ -19,39 +16,33 @@ Example
       -[channel] *120*456#
 
 
-   Favourite Colour
-   ~~~~~~~~~~~~~~~~
+   ### Favourite Colour
```

Comment:
Was also a fan of the underlining, but I've found that the `#`s give one a better idea of the hierarchy. This seemed the same in testing, it seemed like people didn't easily get an idea of which sequence which block was in when they were busy building a dialogue. Both people I tested also with agreed with the `#` change, so I think we should stick with it. +1 on also allowing underlining later though.